audmath/core/api.py

```diff
@@ -652,6 +652,32 @@ def rms(
     return np.sqrt(np.mean(np.square(x), axis=axis, keepdims=keepdims))
 
 
+def samples(
```

Comment:
Regarding the naming i am also split between `samples()` and something like `number_of_samples()` or `duration_in_samples()`.
However, I would almost always prefer a more meaningful name, but i also see the point regarding the naming convention, so I would leave it up to you.
Could you think of any other possible fictions regarding "samples" which would require a distinction?

Reply:
`duration_in_samples()` would be in line with `duration_in_seconds()`, but another reason I decided against it besides not being in line with `audmath.db()` is that it could lead users to expect that both functions can get the same duration values as input, e.g. `audmath.duration_in_seconds()` can handle `pd.Timedelta` objects besides others, whereas here in `audmath.samples()` I restricted the input to duration in seconds given as float.

Another occurence of `samples` is [audiofile.samples(file)](https://audeering.github.io/audiofile/api/audiofile.samples.html) which returns the length of the audio file in samples.

Reply:
Okay, I see.

> Another occurence of samples is [audiofile.samples(file)](https://audeering.github.io/audiofile/api/audiofile.samples.html) which returns the length of the audio file in samples.

Perfect, as this would also be in line with the current naming, i also agree to call it "samples".